cloud-formation/src/templates/cfn-template.yaml

```diff
@@ -16,6 +16,9 @@ Parameters:
   OphanRole:
     Type: String
     Description: ARN of the Ophan cross-account role
+  KinesisStreamArn:
+    Type: String
+    Description: ARN of the kinesis stream to write events to
```

Comment:
is that "acquisition events" or also other types of events?

Reply:
just acquisition